pkg/operator/ceph/cluster/crash/crash.go

```diff
@@ -46,7 +46,7 @@ import (
 
 const (
 	crashCollectorKeyringUsername = "client.crash"
-	crashCollectorSecretName      = "rook-ceph-crash-collector-keyring"
+	crashCollectorSecretName      = "rook-ceph-crash-collector-keyring" // #nosec since gosec flags the variable as a potential problem because of the word `Secret`
```

Comment:
How about if we rename it to `crashCollectorKeyName`? Then we don't need `#nosec`